dev/TabView/TabView.xaml

```diff
@@ -683,23 +709,87 @@
 
                             <Button x:Name="CloseButton"
                                 Grid.Column="2"
-                                Width="{ThemeResource TabViewItemHeaderCloseButtonSize}"
-                                Height="{ThemeResource TabViewItemHeaderCloseButtonSize}"
-                                FontSize="{ThemeResource TabViewItemHeaderCloseFontSize}"
                                 Margin="{ThemeResource TabViewItemHeaderCloseMargin}"
                                 Content="&#xE711;"
                                 IsTextScaleFactorEnabled="False"
                                 IsTabStop="False"
-                                Background="{ThemeResource TabViewItemHeaderCloseButtonBackground}"
-                                Foreground="{ThemeResource TabViewItemHeaderCloseButtonForeground}"
-                                Style="{StaticResource TabViewCloseButtonStyle}"
+                                Style="{ThemeResource TabViewCloseButtonStyle}"
                                 HighContrastAdjustment="None" />
                         </Grid>
                     </Grid>
-
                 </ControlTemplate>
             </Setter.Value>
         </Setter>
     </Style>
 
+    <Style x:Name="TabViewScrollButtonStyle" TargetType="RepeatButton">
+        <Setter Property="Background" Value="{ThemeResource TabViewButtonBackground}"/>
+        <Setter Property="Foreground" Value="{ThemeResource TabViewButtonForeground}"/>
+        <contract7Present:Setter Property="CornerRadius" Value="{ThemeResource ControlCornerRadius}"/>
+        <Setter Property="FontSize" Value="{ThemeResource TabViewItemScrollButonFontSize}"/>
+        <Setter Property="FontFamily" Value="{ThemeResource SymbolThemeFontFamily}"/>
+        <Setter Property="Width" Value="{ThemeResource TabViewItemScrollButtonWidth}"/>
+        <Setter Property="Height" Value="{ThemeResource TabViewItemScrollButtonHeight}"/>
+        <Setter Property="FontFamily" Value="{ThemeResource SymbolThemeFontFamily}"/>
+        <Setter Property="FocusVisualMargin" Value="-3"/>
+        <Setter Property="Template">
```

Comment:
>Template [](http://example.com/codeflow?start=26&length=8)

What is the new template needed for?

Reply:
for correct foreground/background colors in visual states:)

Reply:
We should try not to retemplate, it makes our lives harder in a lot of ways.  That being said, it looks like you removed the default template's BackgroundTransition. I think the only way to do that is via a retemplate, so I think this is alright.